src/main/scala/com/amazon/deequ/analyzers/catalyst/StatefulHyperloglogPlus.scala

```diff
@@ -59,7 +59,7 @@ private[sql] case class StatefulHyperloglogPlus(
 
   override def dataType: DataType = BinaryType
 
-  override def aggBufferSchema: StructType = StructType.fromAttributes(aggBufferAttributes)
+  override def aggBufferSchema: StructType = DataTypeUtils.fromAttributes(aggBufferAttributes)
```

Comment:
`StructType.fromAttributes` has been refactored into `DataTypeUtils.fromAttributes`
Reference: https://github.com/apache/spark/commit/8ff6b7a04cbaef9c552789ad5550ceab760cb078

Reply:
This looks good, it would be great if we could approve and merge this so that we could use it with V3.5